index.js

```diff
@@ -115,11 +116,20 @@ class SlackNotifier extends NotificationBase {
             `${buildData.event.commit.message.substring(0, cutOff)}...` :
             buildData.event.commit.message;
         const isMinimized = buildData.settings.slack.minimized;
-        const message = isMinimized ?
+
+        let message = isMinimized ?
             // eslint-disable-next-line max-len
             `<${pipelineLink}|${buildData.pipeline.scmRepo.name}#${buildData.jobName}> *${buildData.status}*` :
             // eslint-disable-next-line max-len
             `*${buildData.status}* ${STATUSES_MAP[buildData.status]} <${pipelineLink}|${buildData.pipeline.scmRepo.name} ${buildData.jobName}>`;
+
+        const metaMessage = hoek.reach(buildData,
+            'build.meta.notification.message', { default: false });
```

Comment:
Since this is specific for slack can we namespace it under `notification.slack.message` ?

This will be like the tests one we added https://docs.screwdriver.cd/user-guide/metadata#coverage-and-test-results

Reply:
There are some notification tools for us, so it is reasonable. I fixed.